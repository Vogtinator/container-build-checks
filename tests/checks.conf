[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
