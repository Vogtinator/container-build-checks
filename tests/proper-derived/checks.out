Looking at proper-derived.containerinfo
Tag: opensuse/example:latest
Tag: opensuse/example:5.7
Tag: opensuse/example:5.7.7.21
Release: 7.21
Detected image specific label prefix: org.opensuse.example

container-build-checks done. Hints: 0 Warnings: 0 Errors: 0
Exited with 0
